sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -82,9 +82,10 @@ public string CreateTransactionPortfolio(string scope)
                 },
                 transactionDate: tradeDate,
                 settlementDate: tradeDate,
-                units: units,
-                transactionPrice: new TransactionPrice(price, TransactionPrice.TypeEnum.Price),
-                totalConsideration: new CurrencyAndAmount(price*units, currency),
+                units: units ?? default(decimal),
+                transactionPrice: new TransactionPrice(price ?? default(decimal), TransactionPrice.TypeEnum.Price),
+                totalConsideration: new CurrencyAndAmount((
+                        price ?? default(decimal)) * (units ?? default(decimal)), currency),
```

Comment:
Why did you add these defaults in?

Reply:
Should I do `GetValueOrDefault()` instead? 

<img width="715" alt="2021-05-21_10-02-00" src="https://user-images.githubusercontent.com/13220/119112061-99abf980-ba1b-11eb-9d61-45490ffd8ec2.png">

Reply:
Instead, you should make the arguments in `BuildTransactionRequest` non-nullable by removing the `?`

Reply:
Remove units.Value as well and we are good!